content/v2/registrar.markdown

```diff
@@ -76,6 +76,12 @@ Name | Type | Description
 `auto_renew` | `bool` | Set to true to enable the auto-renewal of the domain. Default: `true`.
 `extended_attributes` | `hash` | **Required** for TLDs that require [extended attributes](/v2/tlds/#extended-attributes).
 
+<note>
+The `registrant_id` can be obtained via the [contacts endpoint](/v2/contacts) and will be the registered contact for this
+domain. It is named differently due to the domain registration context technically allowing a registrant, technical, and
```

Comment:
@martinisoft I'd skip the second part as this is not something we are planning anytime soon (from _It is named differently_).